src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 300ms;
```

Comment:
might be a good idea to create a variable, as usually the transition duration is the same across multiple elements